test/commands/generate/models/csharp/AnonymousSchema_1.cs

```diff
@@ -0,0 +1,29 @@
+namespace 'asyncapi.models'
```

Comment:
@peter-rr with your latest commits (In particular [this](https://github.com/asyncapi/cli/pull/859/commits/ea19157a1aadc8a20df0867fb6b6df0855dd268e)) you brought a lot of changes that shouldn't belong to this branch and PR. I don't really get why those changes are here if, in theory, you did a merge from `master` branch to here. 

I suggest we schedule a meeting call so we can pair and fix this together and see what can we do next to try avoid this in the future. Does it sounds 👍 to you?

Reply:
We had a call and finally got fixed.